### ISSUE TITLE
Add option to disable performance issues

### ISSUE THREAD

**Issue by rodolfoBee** [getsentry/sentry#47010]
### Problem Statement

Performance issues are enabled by default and cannot be disabled. Some users want the option to disable them, which is not currently possible. 

### Solution Brainstorm

Add the option to disable performance issues:
- Individually: for example, only Uncompressed Asset
- All Performance issues.

**Comment by souredoutlook** [comment]
Please see linked Jira issue for org breakdown and reasons associated with historical requests for this feature.

**Comment by mjq-sentry** [comment]
This is something we've talked a lot about internally - thanks for the ticket and background info on this, we'll use that context to make some decisions about this.

**Comment by sbv-csis** [comment]
We have a worker that fetches a lot of data from an API that uses `next` paging - aka. For each result we get we get a token to use for the next http call to continue fetching results -> paging.
It consistently end up with Consecutive HTTP because that's exactly the idea :)
It would be nice to have a way to either tell sentry that this is paging - I think I see parts of the code that does this for a specific kind of paging - or a way to instruct sentry to ignore specific consecutive http pieces of code :shrug:  - right now we just use the `ignore` button but I always worry that we end up ignoring "more" than the exact place when we use the interface buttons

**Comment by Bardiamist** [comment]
Same problem https://github.com/getsentry/sentry-react-native/issues/3072

**Comment by chadwhawkins** [comment]
Also interested in this feature

**Comment by nanoburd** [comment]
@chadwhawkins what issue is giving you trouble that the ignore feature doesn't help with?

**Comment by nanoburd** [comment]
Yes, perhaps we can be more clear about this. Performance issues do NOT take up any error quota or additional transaction quota even.

This are free insights that we are generating from the transaction data you're already sending us. So instead of manually going to Performance and "discovering" these problems manually. We are automatically bubbling it up in your feed! =)

And actually we're investing even more resources to surface up even more insights out of your transaction/profiling data to you in the future. Cause who wants to dive through a dashboard?

**Comment by chadwhawkins** [comment]
That's great, thanks @nanoburd. I will proceed to ignore.

**Comment by gnprice** [comment]
My Sentry dashboard is now full of "Consecutive HTTP" items. Every one of them that I've looked at is a false positive — it's there because the app does a long-poll. For us this feature is bringing no value at all, only noise, and I would like to disable it.

Ignoring these one by one as they get generated can help as a workaround, but it means playing whack-a-mole with this feature forever. I would much rather disable it once and for all.

Was this issue thread closed because the option has been implemented (hooray!) and just isn't yet in the UI I see, or because of a decision not to offer a way to disable these? If the latter, I hope you'll reconsider.

**Comment by rodolfoBee** [comment]
@gnprice under Project Settings -> Performance you can increase the threshold used to trigger a new performance issue. 

You can also reach out to us at support@sentry.io with your organisation slug for us to manually disable it from our side.
